source/ftrack_connect_pipeline_3dsmax/asset/dcc_object.py

```diff
@@ -203,7 +202,5 @@ def connect_objects(self, objects):
             if node.Parent == root_node or not node.Parent:
                 node.Parent = dcc_object
                 self.logger.debug(
-                    'Node {} added to dcc_object {}'.format(
-                        node, dcc_object
-                    )
-                )
\ No newline at end of file
+                    'Node {} added to dcc_object {}'.format(node, dcc_object)
+                )
```

Comment:
I think this changes shouldn't be in this PR, may collide later on when trying to merge to story.